pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -140,6 +143,36 @@ func (pp *proportionPlugin) OnSessionOpen(ssn *framework.Session) {
 		if job.PodGroup.Status.Phase == scheduling.PodGroupInqueue {
 			attr.inqueue.Add(job.GetMinResources())
 		}
+		if job.PodGroup.Status.Phase == scheduling.PodGroupRunning && job.PodGroup.Spec.MinResources != nil {
```

Comment:
Should we also consider `scheduling.PodGroupInqueue`?

Reply:
What do you mean? The code above has already added `minResource` of Inqueue jobs into `queue.inqueue`.

Reply:
If the phase of podgroup is `Inqueue`, all the `minResource` has already been taken into `attr.inqueue`.

Reply:
got it.